deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml

```diff
@@ -79,10 +79,13 @@ spec:
   graphing:
     enabled: false
     grafana:
-      ingressEnabled: false
+      ingressEnabled: true
       adminPassword: secret
       adminUser: root
       disableSignoutMenu: false
+      baseImage: registry.redhat.io/rhel8/grafana:7
```

Comment:
Using this as an opportunity to update our base image to use something closer to being supported, and with CVE update tracking.